christian-galarza/lesson-1/contacts_app/src/main/Main.java

```diff
@@ -0,0 +1,29 @@
+package main;
+ /**
+ * This class is start the program.
+ *
+ *  @author Jose Christian Galarza Crespo
+ *  @version 1.0.
+ *
+ */
+ class Main {
+    public static void main(String [] args){
+        ContactsManager myContactsManager = new ContactsManager();
+        Contact friendJames = new Contact();
+         friendJames.name = "James";
```

Comment:
As part of good practices we have to keep the indentation clear.
Please fix the indentation